tests/lib/rss.test.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * RSS Library Test
+ */
+
+const fs = require('fs')
+const rss = require('../../lib/rss')
+const mockdate = require('mockdate')
+
+describe('lib.rss', () => {
+  let rssResponse
+
+  beforeEach(() => {
+    mockdate.set('2017-12-22T08:44:17.000Z')
+
+    rssResponse = rss({
+      data: JSON.parse(
+        fs.readFileSync(`${__dirname}/../__data__/data.json`)
+      ),
+      data_old: JSON.parse(
+        fs.readFileSync(`${__dirname}/../__data__/data_old.json`)
+      ),
+      data_updated: new Date('2017-12-22T08:44:01.327Z'),
+    })
+  })
+
+  afterAll(() => {
+    mockdate.reset()
```

Comment:
If you still use `beforeEach`, then change this to `afterEach`.
It doesn't make sense to mock the date everytime but only reset it once.